infrasec/aws/sns-guardduty-alert-integrations.md

```diff
@@ -229,7 +250,8 @@ resource "pagerduty_service_integration" "guardduty" {
 ### Subscription
 
 ```hcl
-resource "aws_sns_topic_subscription" "team_notification_region" {
+resource "aws_sns_topic_subscription"
+"guardduty_team_notification_region" {
```

Comment:
Is this intentionally on it's own line instead of with the `resource` line above?

Reply:
:) No